driver/ppm_fillers.c

```diff
@@ -926,6 +926,11 @@ bool ppm_is_upper_layer(struct file *exe_file){
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 18, 0)
 	struct super_block *sb = NULL;
 	unsigned long sb_magic = 0;
+	struct ovl_entry *oe = NULL;
```

Comment:
all these initializations at the beginning would cause other warnings on other kernel versions, so I would avoid it

Reply:
according to the `ifdef` some of them could become unused variables